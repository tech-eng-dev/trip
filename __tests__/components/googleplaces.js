import React from 'react';
import renderer from 'react-test-renderer';
import GooglePlacePicker from '../../src/components/googleplaces';


test('GooglePlacePicker renders correctly', () => {
  const googlePicker = renderer.create(<GooglePlacePicker
    onPress={() => {}}
    width={300}
    defaultValue="2018-02-14"
  />).toJSON();
  expect(googlePicker).toMatchSnapshot();
});
