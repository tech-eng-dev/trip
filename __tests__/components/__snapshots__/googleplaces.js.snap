// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GooglePlacePicker renders correctly 1`] = `
<Component
  autoFocus={false}
  fetchDetails={true}
  getDefaultValue={[Function]}
  minLength={2}
  onPress={[Function]}
  placeholder="Enter Location"
  query={
    Object {
      "key": "AIzaSyCxTEst__Vbg0waRjgtd3dOdDESrTOTZPc",
      "language": "en",
    }
  }
  returnKeyType="default"
  styles={
    Object {
      "listView": Object {
        "backgroundColor": "#3DA0F1",
        "padding": 0,
        "width": 300,
      },
      "predefinedPlacesDescription": Object {
        "color": "#1faadb",
      },
      "textInput": Object {
        "backgroundColor": "transparent",
        "color": "#5C6063",
        "fontSize": 16,
        "height": 38,
        "margin": 0,
        "padding": 10,
      },
      "textInputContainer": Object {
        "backgroundColor": "#EEEEEE",
        "borderRadius": 4,
        "borderWidth": 1,
        "padding": 0,
        "width": 300,
      },
    }
  }
/>
`;
